modules/ROOT/pages/delta-sync.adoc

```diff
@@ -37,7 +37,7 @@ Replication does not use Delta Sync when pushing to a pre-2.8 target.
 
 == Configuration
 
-You can enable delta-sync on a per-database basic in your Sync Gateway configuration file using the `delta_sync` properties in{rest-api-admin--xref--database-configuration} -- as shown in <<sample-cfg>>:
+You can enable delta-sync on a per-database basic in your Sync Gateway configuration file using the `delta_sync` properties in {rest-api-admin--xref--database-configuration} -- as shown in <<sample-cfg>>:
```

Comment:
Same comment as before, could we remove the attribute in favour of the inlined xref if it's not used everywhere please